src/common/sms/smsUtils.js

```diff
@@ -0,0 +1,50 @@
+import { Buffer } from 'buffer';
+import { getCrypto } from './crypto.js';
+
+export const formatSecretValue = async (secretValue) => {
```

Comment:
maybe rename this method `formatBeneficiaryKey` as this is specific for beneficiary key